docs/tutorials/podman_tutorial_cn.md

```diff
@@ -0,0 +1,238 @@
+                  registry.fedoraproject.org/f29/httpd /usr/bin/run-httpd
+```
+
+
+
+
+因为命令中的`-d`参数表明容器以`detached`模式运行，所以Podman会在容器运行后打印容器的ID。
+
+注意为了访问这个HTTP服务器，我们将使用端口转发。成功运行需要slirp4nextns 的 v0.3.0+版本。
+
+
+Podman的`ps`命令用于列出正在创建和运行的容器。
+
+```console
+podman ps
+```
+
+
+
+
+**注意**：如果为`ps`命令添加 `-a`参数，Podman 将展示所有的容器。 
+
+
+
+### 查看正在运行的容器
+
+
+
+你可以`inspect` (查看)一个正在运行的容器的元数据以及其他详细信息。我们甚至可以使用 `inspect`的子命令查看分配给容器的IP地址。由于容器以非root模式运行，没有分配IP地址，`inspect`的输出会是 none 
+
+```console
+podman inspect -l | grep IPAddress\":
+            "SecondaryIPAddresses": null,
+            "IPAddress": "",
+```
+
+**注意**：`-l` 参数是**最近的容器**的指代，你也可以使用容器的ID 代替`-l`
+
+
+### 测试httpd服务器
+
+
+
+由于我们没有容器的IP地址，我们可以使用curl测试主机和容器之间的网络通信。下面的命令应该显示我们的容器化httpd服务器的主页。
+
+```console
+curl http://localhost:8080
+```
+
+
+### 查看容器的日志
+
+
+你也可以使用podman查看容器的日志
+
+```console
+podman logs --latest
+10.88.0.1 - - [07/Feb/2018:15:22:11 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.55.1" "-"
+10.88.0.1 - - [07/Feb/2018:15:22:30 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.55.1" "-"
+10.88.0.1 - - [07/Feb/2018:15:22:30 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.55.1" "-"
+10.88.0.1 - - [07/Feb/2018:15:22:31 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.55.1" "-"
+10.88.0.1 - - [07/Feb/2018:15:22:31 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.55.1" "-"
+```
+
+
+### 查看容器中的进程pid
+
+
+你可以使用`top`命令查看容器中httpd的pid
+
+```console
+podman top <container_id>
+  UID   PID  PPID  C STIME TTY          TIME CMD
+    0 31873 31863  0 09:21 ?        00:00:00 nginx: master process nginx -g daemon off;
+  101 31889 31873  0 09:21 ?        00:00:00 nginx: worker process
+```
+
+
+
+### 设置容器的检查点
+
+
+
+设置检查点会在停止容器的同时把容器中所有进程的状态写入磁盘。
+
+有了它，容器后续可以被恢复，并在与检查点完全相同的时间点继续运行。
+这个功能需要在系统上安装CRIU 的3.11+版本。
+
+这个功能不支持非 root 模式，因此，如果你想尝试使用它，你需要使用`sudo`方式重新创建容器。
+
+
+设置容器检查点请使用：
+
+```console
+sudo podman container checkpoint <container_id>
+```
+
+
+### 恢复容器
+
+
+恢复容器只能在以前设置过检查点的容器上使用。恢复的容器会在与设置检查点时完全相同的时间点继续运行。
+
+恢复容器请使用：
+
+```console
+sudo podman container restore <container_id>
+```
+
+
+
+
+恢复之后。容器会像设置检查点之前一样回复请求
+
+```console
+curl http://<IP_address>:8080
+```
+
+
+
+### 迁移容器
+
+
+
+为了将容器从一个主机上热迁移到另一个主机，容器在源系统上创建检查点，传输到目的系统，然后再目的系统上恢复。
+
+当传输检查点时，可以指定一个输出文件。
+
+
+在源系统上：
+
+```console
+sudo podman container checkpoint <container_id> -e /tmp/checkpoint.tar.gz
+scp /tmp/checkpoint.tar.gz <destination_system>:/tmp
+```
+
+
+
+在目标系统上：
+
+```console
+sudo podman container restore -i /tmp/checkpoint.tar.gz
+```
+
+
+
+
+恢复之后，容器会像设置检查点之前一样回复请求。这时，容器会在目标系统上继续运行。
+
+```console
+curl http://<IP_address>:8080
+```
+
+
+
+### 停止容器
+
+
+停止httpd 容器
+
+```console
+podman stop --latest
+```
+
+
+
+你还可以使用`ps`命令检查一个或多个容器的状态，在这个例子中，我们使用 `-a` 参数列出所有的容器。
+
+```console
+podman ps -a
+```
+
+
+
+### 移除容器
+
+
+移除httpd容器
+
+```console
+podman rm --latest
+```
+
+
+你可以使用`podman ps -a` 验证容器的删除。
+
+
+## 集成测试
+
+在环境中如何设置并运行集成测试请查看 集成测试的[自述页面](https://github.com/containers/podman/blob/main/test/README.md)
```

Comment:
For a translate file, maybe it is better to keep the link the same as the original one?